sdk/iot/Azure.Iot.Hub.Service/src/IotHubServiceClient.cs

```diff
@@ -21,6 +21,10 @@ public class IotHubServiceClient
         private readonly QueryRestClient _queryRestClient;
         private readonly StatisticsRestClient _statisticsRestClient;
 
+        // IoT Hub service currently does not support OAuth tokens, so they do not have their authorization scopes defined.
+        // This value will need to be correctly populated once OAuth token support is available.
+        private static readonly string[] s_authorizationScopes = new[] { "" };
```

Comment:
This is a weird piece of code that is required for the client to be able to use BearerTokenAuthenticationPolicy